src/components/UserMenu/UserMenu.tsx

```diff
@@ -1,13 +1,12 @@
 import classNames from 'classnames';
-import UserIcon from 'images/icons/user.svg';
 import { useEffect, useState } from 'react';
 import { NavLink } from 'react-router-dom';
 import { AppRoutes } from 'types/AppRoutes';
 
 import styles from './UserMenu.module.scss';
 import { UserMenuProps } from './UserMenu.types';
 
-export function UserMenu({ userName, className }: UserMenuProps) {
+export function UserMenu({ userName, className, image }: UserMenuProps) {
```

Comment:
❗️
Кажется, тут нет смысла передавать объект, так как заешь ты всё равно только `src`
```suggestion
export function UserMenu({ userName, className, imageSrc }: UserMenuProps) {
```

Reply:
Подскажи, плиз, почему тут решил оставить?